### DISCUSSION TITLE
Doge Block Explorer

### DISCUSSION THREAD

**Discussion by vasuxdev** [dogecoin/dogecoin#3247] [Help needed]
Is there any github repository to deploy my own block explorer with Dogecoin core node?

**Comment by iamtiagogalvao** [comment]
Deploying your own block explorer for Dogecoin or any other cryptocurrency involves running a full node and setting up additional software to parse and display the blockchain data in a user-friendly manner. There are a few open-source block explorer projects on GitHub that you might adapt for Dogecoin. Note that these may require some adjustments to work with Dogecoin specifically.

Here is an option you might start with:
[Iquidus Explorer](https://github.com/iquidus/explorer)
